### PR TITLE
Update with-exchange-online.md

### PR DESCRIPTION
**Pull request by pupara**
Added support for TPM 2.0

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/with-exchange-online.md [hunk 1]
```diff
@@ -120,10 +120,11 @@ If you deployed Active Directory Federation Services (AD FS), you may have to co
 > If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
 
     ``` Powershell
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-exchange-online.md:122]:
```suggestion
When using Teams PowerShell Module:
    ``` Powershell
```

File: Teams/rooms/with-exchange-online.md [hunk 2]
```diff
@@ -120,10 +120,11 @@ If you deployed Active Directory Federation Services (AD FS), you may have to co
...
-    Import-Module -Name MicrosoftTeams
-    $cred = Get-Credential
-    $cssess = New-CsOnlineSession -Credential $cred  
-    Import-PSSession $cssess -AllowClobber
+    # When using Teams PowerShell Module
+
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-exchange-online.md:124]:
```suggestion
```

Comment (JohanFreelancer9) [inline Teams/rooms/with-exchange-online.md:124]:
```suggestion
```

File: Teams/rooms/with-exchange-online.md [hunk 3]
```diff
@@ -120,10 +120,11 @@ If you deployed Active Directory Federation Services (AD FS), you may have to co
 > If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
 
     ``` Powershell
-    Import-Module -Name MicrosoftTeams
-    $cred = Get-Credential
-    $cssess = New-CsOnlineSession -Credential $cred  
-    Import-PSSession $cssess -AllowClobber
+    # When using Teams PowerShell Module
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-exchange-online.md:123]:
```suggestion
```

File: Teams/rooms/with-exchange-online.md (hunk 2 above)
Comment (JohanFreelancer9) [inline Teams/rooms/with-exchange-online.md:124]:
```suggestion
```

File: Teams/rooms/with-exchange-online.md [hunk 4]
```diff
@@ -120,10 +120,11 @@ If you deployed Active Directory Federation Services (AD FS), you may have to co
...
+
+   Import-Module MicrosoftTeams
+   $credential = Get-Credential
+   Connect-MicrosoftTeams -Credential $credential
     ```
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-exchange-online.md:128]:
```suggestion
   ```
```

Comment (JohanFreelancer9) [inline Teams/rooms/with-exchange-online.md:128]:
```suggestion
   ```
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@pupara - the copy is fine but the new PowerShell command does not render correctly. Please have a look at the rich diff view.

**Comment by pupara** [comment]
Fixed...


Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**